UPGRADE.md

```diff
@@ -38,6 +38,16 @@ The `checkCredentials` hook has been removed. Use the `CheckPassportEvent` inste
 
 The `postLogin` hook has been removed. Use the `LoginSuccessEvent` instead.
 
+### postDownload hook
+
+The `postDownload` is triggered in `Controller::sendFileToBrowser()` which has been deprecated. If you e.g. relied on
+the `postDownload` hook being called in the download(s) content elements, this is no longer the case unless you are
+actively using the legacy content elements as shown later in this document. Replace your hook with a general response
+event listener listening to the `ResponseEvent` and check if it's of instance `BinaryFileResponse`. If so, you can use
+`BinaryFileResponse::getFile()` in order to access the file and apply your business logic. This has the advantage that
+it works for all responses sent across the system, not just for the ones using the deprecated
+`Controller::sendFileToBrowser()` method.
+
```

Comment:
```suggestion

```

That is the wrong place for this information. The `UPGRADE.md` file lists stuff that has already been removed, hence the headline _Version 4.* to 5.0_.

Reply:
I put it here because this is the place where I searched why the `postDownload` hook is not called anymore. If you have an extension in Contao 4 that uses this hook, it won't work anymore in Contao 5. It has not been removed but effectively it has because the new content elements don't call the hook anymore :)

Reply:
I see 👍